### DISCUSSION TITLE
Could CHD support MSU-1?

### DISCUSSION THREAD

**Discussion by jason-oliveira** [mamedev/discussions#56] [Ideas]
I appreciate this might be a few layers of question, as I'm unsure as to the current support of Byuu/Near's MSU-1 spec in MAME, or whether it could ever even *be* supported due to the lack of retail releases. 

Seeing as MSU-1 is just a series of uncompressed pcm tracks alongside a rom and a .msu/.xml file, how much would the CHD format/chdman need to be extended to support this? Is this a doable-but-difficult task, or is this a square-peg-round-hole issue? For end-users, being able to just store and load a .chd file is preferable to a >1GB folder with a .sfc file to hunt for.

**Comment by rb6502** [comment]
This is doable, but we'd need MSU-1 support in MAME's SNES emulation first.
